shotover-proxy/src/message/mod.rs

```diff
@@ -182,18 +182,21 @@ impl Message {
         }
     }
 
-    pub fn message_count(&self) -> Result<NonZeroU32> {
+    /// Get the count of "cells" in this message. In Shotover, all messages are only 1 cell except for batch messages which can contain 1 or more cells.
+    /// A cell in a batch message is equivalent to one of it's inner batched messages.
```

Comment:
I find this sentence hard to understand.

We could write it more simply as:
```
Each inner message in a batch message is worth one cell.
```

Or maybe a bullet list of cell count logic would be more clear than a paragraph:
```
Cell count:
* Regular message - 1 cell
* Message containing submessages e.g. a batch request - 1 cell per submessage
* Message containing submessages with 0 submessages - 1 cell
```

Reply:
The idea of the bullet points is to supercede all count explanations in the paragraph.
So we would get rid of "In Shotover, all messages are only 1 cell except for batch messages which can contain 1 or more cells. Batch messages have a cell count of 1 cell per inner message. "

Is that alright?

Reply:
Yes, I prefer your way of explaining things with the points. I'll switch it over.